docker/include/network/scripts/new_ip

```diff
@@ -36,4 +36,7 @@ else
     new="dhcp-host=$mac_addr,$new_ip"
     flock /etc/dnsmasq.conf echo $new >> /etc/dnsmasq.conf
 fi
+
+source change_dhcp_range $(echo $new_ip | cut -d '.' -f 1,2,3).0 $(echo $new_ip | cut -d '.' -f 1,2,3).255 24
```

Comment:
@henry54809 Looking back at the test_dhcp runs on master, it looks as if the new_ip script does not work if used alongside the multi-subnet test. This is because the device is sending a DHCP request which is accepted by dnsmasq (even though a static host has been set) because the static host is outside of any range registered in the dnsmasq config. I tried adding a static range like this: `dhcp-range=10.0.0.0,static` but this was not successful. The current method is successful however by changing the whole range.

Reply:
Can you please revert the changes here. I will fix this in a different PR.